.github/workflows/ci.yaml

```diff
@@ -0,0 +1,33 @@
+name: Wrangler CI
+
+on:
+  push:
+  pull_request:
+    tags:
+      - v*
+    branches:
+      - 'release/*'
```

Comment:
The branches are named differently for wrangler. The currently supported branches are:
- `master`
- `release-2.2`
- `release-2.1`
- `release-2.0-untagged`

```suggestion
      - 'release-*'
```

Reply:
Good catch